### PR TITLE
[TACHYON-1510] Start and stop Tachyon asynchronously

### PR DESCRIPTION
**Pull request by caadecarvalho**
Changed the start/stop from bin/tachyon-workers.sh to be performed asynchronously.

### CODE REVIEW

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

File: bin/tachyon-workers.sh [hunk 1]
```diff
@@ -19,18 +19,20 @@ DEFAULT_LIBEXEC_DIR="${BIN}"/../libexec
 TACHYON_LIBEXEC_DIR=${TACHYON_LIBEXEC_DIR:-$DEFAULT_LIBEXEC_DIR}
 . $TACHYON_LIBEXEC_DIR/tachyon-config.sh
 
-HOSTLIST=$TACHYON_CONF_DIR/workers
+HOSTLIST=$(cat ${TACHYON_CONF_DIR}/workers | sed  "s/#.*$//;/^$/d")
+TACHYON_TASK_LOG="$(echo ${BIN} | sed 's/bin$//g')"logs/task.log
```
Comment (yupeng9) [inline bin/tachyon-workers.sh]: Could you describe what this change is for?

Comment (caadecarvalho) [inline bin/tachyon-workers.sh]: I added a log for the task operation and also a I refactored `HOSTLIST`

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

**Comment by caadecarvalho** [comment]
Hi @yupeng9 , I made a forced update with better comments, please take a look at it.

**Comment by apc999** [comment]
+1 LGTM

**Comment by yupeng9** [comment]
@uronce-cc could you take a look?

**Comment by yupeng9** [comment]
LGTM

**Comment by uronce-cc** [comment]
LGTM

**Comment by yupeng9** [comment]
@caadecarvalho thanks for the improvement!

**Pull request merged**